topology/probes/netlink.go

```diff
@@ -79,6 +79,25 @@ type NetLinkProbe struct {
 	wg      sync.WaitGroup
 }
 
+// RouteTable describes a list of Routes
+type RoutingTable struct {
+	Id     int
```

Comment:
I tried to use multiple tables but it seems that only the default table is reported

Reply:
I thought there will be one table, so I wrote a code according to that.
I feel if you review the "getRoutingTable" method we can verify whether the implementation is correct or not.
In my system, I am getting routes for two table id's but from netlink i am getting only one id for all the routes. PSB the command line output
-bash-4.2$ sudo ip netns exec vm1 ip route show table 255
broadcast 124.65.91.0 dev eth0  proto kernel  scope link  src 124.65.91.42 
local 124.65.91.42 dev eth0  proto kernel  scope host  src 124.65.91.42 
broadcast 124.65.91.255 dev eth0  proto kernel  scope link  src 124.65.91.42 
-bash-4.2$ sudo ip netns exec vm1 ip route show table 254
124.65.91.0/24 dev eth0  proto kernel  scope link  src 124.65.91.42 

from netlink i am getting only 254

Please correct me if i understood and conveyed anything wrong.